server/controllers/stock/reports/stock_entry_donation.receipt.handlebars

```diff
@@ -0,0 +1,107 @@
+{{> head title="STOCK.RECEIPT.ENTRY_DONATION" }}
+
+<!-- body  -->
+<div class="container" style="font-size: 0.9em;">
+
+  <header>
+    <!-- headings  -->
+    <div class="row">
+      <div class="col-xs-6">
+        <h3 style="margin: 0px;">{{enterprise.name}}</h3>
+        <p>
+          <span class="text-capitalize">{{translate 'FORM.LABELS.ADDRESS'}}</span>: {{enterprise.location}} <br>
+          <span class="text-capitalize">{{translate 'FORM.LABELS.PHONE'}}</span>: {{enterprise.phone}} <br>
+          <span class="text-capitalize">{{translate 'FORM.LABELS.EMAIL'}}</span>: {{enterprise.email}}
+        </p>
+      </div>
+      <div class="col-xs-6 text-right">
+        <h3 style="margin: 0px;">
+          <span class="text-uppercase">
+            {{translate 'STOCK.RECEIPT.ENTRY_DONATION'}}
+          </span> <br>
+          <strong>{{ details.depot_name}}</strong> <br>
+          <strong>{{ details.document_reference}}</strong> <br>
+          <small>{{date details.date}}</small> <br>
+          <br>
+        </h3>
+      </div>
+    </div>
+
+    <!-- client and user details  -->
+    <div class="row" style="border: 1px solid #ccc; padding: 5px; margin-bottom: 15px;">
+      <div class="col-xs-6">
+        <h4>{{translate 'STOCK.FROM'}}: {{translate 'STOCK.ENTRY_DONATION'}}</h4>
+        <span class="text-capitalize">{{translate 'FORM.LABELS.DATE'}}</span>: {{date details.date}} <br>
+      </div>
+      <div class="col-xs-6">
+        <h4>{{translate 'STOCK.TO'}}</h4>
+        <span class="text-capitalize">{{translate 'STOCK.DEPOT'}}</span>: <strong>{{details.depot_name}}</strong> <br>
+        <span class="text-capitalize">{{translate 'FORM.LABELS.DOCUMENT'}}</span>: <strong>{{details.document_reference}}</strong> <br>
+        <span class="text-capitalize">{{translate 'FORM.LABELS.DATE'}}</span>: {{date details.date}} <br>
+        <span class="text-capitalize">{{translate 'FORM.LABELS.COST'}}</span>: {{currency (sum rows 'total') enterprise.currency_id}} <br>
+        <span class="text-capitalize">{{translate 'STOCK.INVENTORY'}}</span>: {{rows.length}} {{translate 'STOCK.ITEMS'}} <br>
+        <span class="text-capitalize">{{translate "TABLE.COLUMNS.CREATED_BY"}}</span>: {{details.user_display_name}} <br>
+      </div>
+    </div>
+  </header>
+
+  {{#if details.description}}
```

Comment:
Is `description` allowed to be empty?

Reply:
The `description` is not allowed to be empty on the client side. On the server side empty description is recorded, The problem is if we have an empty description the `blockquote` tag style an empty block : 
```html
<blockquote>
  <strong>{{translate 'FORM.LABELS.DESCRIPTION'}}</strong><br>
  {{details.description}}
</blockquote>
```

Reply:
When would we have an empty description on the server if it isn't allowed on the client?